lib/index.js

```diff
@@ -356,16 +368,95 @@ function Setup(T, { checkTypes , _env=[] }){
 
     const Class =
         def(
-            'UnionType.Class'
+            'SumType.Class'
             ,{}
             ,[T.String, T.StrMap(T.Any), T.Object, T.Any]
-            ,CreateUnionType
+            ,CreateSumType
         )
 
+    const $SumType = T.UnaryType(
+        'JAForbes/SumType'
+        ,''
+        , x => [x]

```

Comment:
Usually I don't mind if spacing is inconsistent (within reason). 

I prefer a vertical style (for various reasons) so I end up with a lot of lists like so:

```js
[ a
, b
, c
]
```

or object literals:

```js
{ a
, b
, c
} 
```

I like the `,` to align with the braces/brackets, and then each value will align, whether there's a space prefix or not.

When the list is within a function call though, I'm a lot less strict because the parens are up on a different line.  I suppose I could technically move the parens down for consistency:

```js
T.UnaryType
  ( 'JAForbes/SumType'
  , ''
  , x => [x]
  )
```

But I'm not sure how I feel from the parens being so far from the function name, I need to reflect on that.  I'm not at all opposed to diverging from normal code style practices if they are consistent and have practical benefits like less merge conflicts (as an example).

I think in the above sample I was iterating on it quite often and there wasn't much rhyme or reason to it.  I just wanted to get the concept out of my head and into code 😄